src/main/java/nl/novi/techiteasy/models/Television.java

```diff
@@ -0,0 +1,202 @@
+package nl.novi.techiteasy.models;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+import java.util.Date;
+
+@Entity
+public class Television {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String type;
+    private String brand;
+    private String name;
+    private double price;
+    private double availableSize;
+    private int refreshRate;
+    private String screenType;
+    private String screenQuality;
+    private Boolean smartTv;
+    private Boolean wifi;
+    private Boolean voiceControl;
+    private Boolean hdr;
+    private Boolean bluetooth;
+    private Boolean ambiLight;
+    private int originalStock;
+    private int sold;
+    private Date lastSold;
```

Comment:
Date is een verouderd type van Java. Het zit enkel nog in de java libraries vanwege backwards compatibility. Wat de mensen van java zelf aanraden om te gebruiken is LocalDate, LocalTime of LocalDateTime. Date wordt namelijk weergegeven als een Unix Timestamp (milliseconden sinds 1-1-1970), terwijl LocalDate gewoon in ISO 8601 (YYYY-MM-DD) wordt weergegeven